website/pages/use-cases/simple-container-orchestration.jsx

```diff
@@ -2,8 +2,30 @@ import UseCasesLayout from 'components/use-case-page'
 import TextSplitWithCode from '@hashicorp/react-text-split-with-code'
 import TextSplitWithImage from '@hashicorp/react-text-split-with-image'
 import FeaturedSliderSection from 'components/featured-slider-section'
+// Imports below are used in getStaticProps only
+import highlightData from '@hashicorp/nextjs-scripts/prism/highlight-data'
 
-export default function SimpleContainerOrchestrationPage() {
+export async function getStaticProps() {
+  const codeBlocksRaw = {
+    containerOrchestration: {
+      code:
+        'task "webservice" {\n  driver = "docker"\n\n  config {\n    image = "redis:3.2"\n    labels {\n      group = "webservice-cache"\n    }\n  }\n}',
+      language: 'hcl',
+    },
+    windowsSupport: {
+      code:
+        'sc.exe start "Nomad"\n\nSERVICE_NAME: Nomad\n      TYPE               : 10  WIN32_OWN_PROCESS\n      STATE              : 4  RUNNING\n                              (STOPPABLE, NOT_PAUSABLE, ACCEPTS_SHUTDOWN)\n      WIN32_EXIT_CODE    : 0  (0x0)\n      SERVICE_EXIT_CODE  : 0  (0x0)\n      CHECKPOINT         : 0x0\n      WAIT_HINT          : 0x0\n      PID                : 8008\n      FLAGS              :',
+    },
```

Comment:
This `windowsSupport` snippet doesn't strictly need to be in `getStaticProps` since it doesn't require highlighting - I've just added it for consistency in how code block feature sections are managed.